worlds/wargroove/Client.py

```diff
@@ -46,6 +46,7 @@ def _cmd_deathlink(self):
         """Toggles deathlink On/Off"""
         if isinstance(self.ctx, WargrooveContext):
             self.ctx.has_death_link = not self.ctx.has_death_link
+            self.ctx.update_death_link(self.ctx.has_death_link)
```

Comment:
since this is a non-async context you would want to do something like
```suggestion
            Utils.async_start(self.ctx.update_death_link(self.ctx.has_death_link), name="Update Deathlink")
```